common/crapto1/crypto1.c

```diff
@@ -47,6 +47,7 @@ struct Crypto1State * crypto1_create(uint64_t key)
 	struct Crypto1State *s = malloc(sizeof(*s));
 	int i;
 
+	memset(s, 0x0, sizeof(*s));
```

Comment:
This is not necessary. The struct is initialized in the following for loop. The memory content after malloc is not relevant.